src/com/github/alex_moon/porcupi/handlers/PokeHandler.java

```diff
@@ -1,25 +1,58 @@
 package com.github.alex_moon.porcupi.handlers;
 
+import java.util.ArrayList;
 import java.util.List;
 
+import com.github.alex_moon.porcupi.contexts.PokeContext;
+import com.github.alex_moon.porcupi.manager.ManagerServer;
+import com.github.alex_moon.porcupi.messages.Message;
+import com.github.alex_moon.porcupi.messages.PokeMessage;
+
 public class PokeHandler implements Handler {
     private Pokeable pokeable;
+    private List<PokeContext> contexts = new ArrayList<PokeContext>();
```

Comment:
Surely any `Handler` has `contexts`? Maybe not - the interface might rather look like `ContextHandler` - but this makes subclassing a lot more difficult and since this is a private variable it doesn't really make any sense to define the interface.